tests/layer_tests/pytorch_tests/test_addmm.py

```diff
@@ -83,8 +83,14 @@ def forward(self, m0, m1, m2):
 
     ])
     @pytest.mark.parametrize("alpha,beta",
-                             [(1., 1.), (0., 1.), (1., 0.), (1., 2.), (2., 1.), (-5., -6.), (3., 4.), (0.5, 0.75),
-                              (1, 1)])
+                             [  # beta==0 in some cases produce nan in pytorch
```

Comment:
Sounds weird. The doc https://pytorch.org/docs/stable/generated/torch.baddbmm.html just says that it should work fine. Pytorch bug?

Reply:
Yeah, looks like pytorch bug, because it works in ov

Reply:
looks regular matrices multiplication. How `nan` values can happen here?